docs/releases/1.9.txt

```diff
@@ -239,6 +239,9 @@ Minor features
 * Added the :class:`~django.contrib.postgres.functions.TransactionNow` database
   function.
 
+* Fixed :class:`~django.contrib.postgres.forms.HStoreField` which has never
+  worked.
+
```

Comment:
Yeah... this wording is WIP [hence the original commit message for it being "started" writing docs]